var/spack/repos/builtin/packages/hydrogen/package.py

```diff
@@ -135,13 +135,12 @@ def cmake_args(self):
         ]
 
         # Add support for OS X to find OpenMP
-        if (self.spec.satisfies('%clang')):
-            if (sys.platform == 'darwin'):
```

Comment:
@bvanessen Since this package no longer uses `sys`, flake is complaining about the `import sys` at the top.

Reply:
Removed it.